extensions/resource-deployment/src/services/tools/azCliTool.ts

```diff
@@ -40,18 +43,99 @@ export class AzCliTool extends ToolBase {
 		return true;
 	}
 
-	public install(): Promise<void> {
-		return Promise.reject('not implemented');
+	protected async getInstallationPath(): Promise<string | undefined> {
+		switch (this.osType) {
+			case OsType.linux:
+				return await Promise.resolve(defaultInstallationRoot);
+			default:
```

Comment:
Just return the strings directly, no need to wrap them

Reply:
Done